src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -3094,13 +3096,15 @@ class C<T>
             // sbyte, byte, short, ushort, int, uint, long, ulong, char, float, double, decimal, or bool.
             // Any enum_type.
             // Any pointer_type.
-            // Any user-defined struct_type that is not a constructed type and contains fields of unmanaged_types only.
+            // Any user-defined struct_type that contains fields of unmanaged_types only.
```

Comment:
@agocke should I send a PR to csharplang to update the spec language over there?